dags/graphics_telemetry.py

```diff
@@ -61,13 +62,15 @@
     else:
         aws_access_key, aws_secret_key, _ = AwsBaseHook(aws_conn_id=write_aws_conn_id, client_type='s3').get_credentials()
 
-    wait_for_main_ping = ExternalTaskCompletedSensor(
+    wait_for_main_ping = ExternalTaskSensor(
         task_id="wait_for_main_ping",
         external_dag_id="copy_deduplicate",
         external_task_id="copy_deduplicate_main_ping",
         execution_delta=datetime.timedelta(hours=2),
         check_existence=True,
         mode="reschedule",
+        allowed_states=['success'],
```

Comment:
The allowed states default to "success", so this isn't technically necessary, but I'm not opposed to being explicit.